### PR TITLE
p10k: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by CleanMachine1** [comment]
So close! 

I made this page with this [program](https://github.com/CleanMachine1/tldr-page-creator) and the only thing that didn't work was the more information, since I hadn't pulled on my branch!

File: pages/common/p10k.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# p10k
+
+> Configuration for powerlevel10k.
+> More information: <https://github.com/romkatv/powerlevel10k>.
+
+- Interactively configure a powerlevel10k:
```
Comment (navarroaxel) [inline pages/common/p10k.md:6]: We prefer start with the verb
```suggestion
- Configure a powerlevel10k interactively:
```

File: pages/common/p10k.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# p10k
+
+> Configuration for powerlevel10k.
```
Comment (navarroaxel) [inline pages/common/p10k.md:3]: Is this better?
```suggestion
> Manage configurations for powerlevel10k.
```

File: pages/common/p10k.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/p10k.md:6]: edited to just remove the `a`

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**